narwhals/utils.py

```diff
@@ -1846,7 +1828,7 @@ def _ensure_version(self, instance: _FullContext, /) -> None:
         if instance._backend_version >= self._min_version:
             return
         method = self._wrapped_name
-        backend = instance._implementation._alias
+        backend = instance._implementation
```

Comment:
Should we do 
```suggestion
        backend = str(instance._implementation)
```
to be more explicit?

Reply:
So in (#2371) I only split it up that way for readability.

If you can get the same output with `str(...)`, then you could use `!s`

```py
msg = f"`{method}` is only available in {backend!s}>={minimum!r}, found version {found!r}."
```

https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals